java/src/jmri/jmrit/display/layoutEditor/LayoutBlock.java

```diff
@@ -105,19 +112,19 @@ public void disableDeleteRouteLog() {
     ResourceBundle rb = ResourceBundle.getBundle("jmri.jmrit.display.layoutEditor.LayoutEditorBundle");
 
     //constants
-    public static final int OCCUPIED = jmri.Block.OCCUPIED;
-    public static final int EMPTY = jmri.Block.UNOCCUPIED;
-    public static final int UNKNOWN = jmri.Sensor.UNKNOWN;  //must be a different bit
+    public static final int OCCUPIED = Block.OCCUPIED;
+    public static final int EMPTY = Block.UNOCCUPIED;
+    public static final int UNKNOWN = Sensor.UNKNOWN;  //must be a different bit
     //operational instance variables (not saved to disk)
     private int useCount = 0;
     private NamedBeanHandle<Sensor> occupancyNamedSensor = null;
     private NamedBeanHandle<Memory> namedMemory = null;
 
-    //private jmri.Memory memory = null;
-    private jmri.Block block = null;
+    //private Memory memory = null;
+    private Block block = null;
 
     //private int maxBlockNumber = 0;
-    private LayoutBlock _instance = null;
+    //private LayoutBlock _instance = null; //TODO: delete this
```

Comment:
I just wanted to see if anything blew up before I permanently removed it. Although privates are safer to remove than protected or publics I'm still somewhat paranoid about removing things until I know everything it may break. (I'll usually comment it out first and then fix everything that used it… and then if all tests pass go back and delete it. Which I can do now. Fixed.)

Reply:
(Basically the "TODO:" is a reminder to clean up after myself. ;-)